src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -129,9 +135,9 @@ export default function ProjectSnippet({
                   {getFormatedCurrency(
                     i18n.language,
                     project.currency,
-                    project.treeCost
+                    project.unitCost
```

Comment:
`.treeCosts` is still widely used within our code, e.g. at line 118 above this or in `PopupProject.tsx` - so I do not understand why we want to change it only right here now? For me the code get inconsistent this way and I won't change it in this PR if there is no reason to do so.

Reply:
replaced treeCost with unitCost in remaining places.